### PR TITLE
tests: setup integration tests

### PR DESCRIPTION
**Pull request by erizocosmico**
Dunno how useful will be running tests from SQL queries (specially after the iterators refactor @ajnavarro is doing) but... here it is.

### CODE REVIEW

File: src/integration/scala/tech/sourced/api/IntegrationTest.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  var ss: SparkSession = _
+  val resourcesPath = getClass.getResource("/siva-files").toString
+
+  "SQL queries" should {
+    Source.fromInputStream(getClass.getResourceAsStream("/queries.sql"))
+      .getLines
+      .foreach(sql => s"query: $sql" in {
+        val df = ss.sql(sql)
+        println(sql)
+        df.show
```
Comment (ajnavarro) [inline src/integration/scala/tech/sourced/api/IntegrationTest.scala]: we should check the output result too

Comment (erizocosmico) [reply src/integration/scala/tech/sourced/api/IntegrationTest.scala]: Then we'd need to move away from a plain SQL file and use something more complex (or add a scala test file for it)

**Comment by erizocosmico** [comment]
I changed the SQL file with a `Map` to also be able to check the results. I don't know how maintainable will that be in the future if we start adding a lot of queries.

Also, I tried running these tests with other bigger siva files and it was awfully slow, so I had to resort to use the same files used right now in regular unit tests until we have the iterator fix or something.

**Comment by erizocosmico** [comment]
Shall we merge or close this?

**Comment by erizocosmico** [comment]
ping @ajnavarro

**Comment by erizocosmico** [comment]
Rebased, merging

**Comment by erizocosmico** [comment]
This is hella outdated and has a previous build config so I will close this and will put up a new PR with this once we need it with updated build conf and so on.

**Pull request closed without merging**